<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recognizebyen_form</class>
 <widget class="QWidget" name="recognizebyen_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>按英文背中文</string>
  </property>
  <widget class="QLabel" name="word_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>认识</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>不认识</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tooeasy_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>太简单</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tips_tb">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>600</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="playaudio_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>120</y>
     <width>75</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>读音</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
